typo3/cms-core/2020-11-17-1.yaml

```diff
@@ -0,0 +1,13 @@
+title: 'TYPO3-CORE-SA-2020-009: Cross-Site Scripting through Fluid view helper arguments'
+link: 'https://typo3.org/security/advisory/typo3-core-sa-2020-009/'
+branches:
+    10.x:
+        time: '2020-11-17 08:50:35'
+        versions: ['>=10.0.0', '<10.4.10']
+    9.x:
+        time: '2020-11-17 08:49:44'
+        versions: ['>=9.0.0', '<9.5.23']
+    8.x:
+      time: '2020-11-17 08:55:33'
+      versions: ['>=8.7.0', '<8.7.38']
+reference: 'composer://typo3/cms-core'
```

Comment:
As the issue happens in `typo3fluid/fluid`, and that `typo3/cms-core` does not depend on an exact version of fluid, this does not guarantee that the advisory matching is right for projects using composer.
Instead, the advisory should be assigned to the `typo3fluid/fluid` package itself

Reply:
`typo3fluid/fluid` is totally separate thing... however `typo3/cms` has strong dependencies on it as well.

→ see Fluid dependency https://github.com/TYPO3/TYPO3.CMS/blob/master/composer.json#L80
→ at least `2.6.10` having the latest security fixes included

Reply:
Yeah, but as the security issue is in typo3fluid/fluid (at least according to my understanding of the linked page), the advisory should be on that package, not on `typo3/cms` and `typo3/cms-core` which depend on it.
Bundled versions of typo3fluid/fluid in the typo3 CMS full download are not meaningful here, because composer-based installation are not relying on such bundled dependencies but on composer resolving dependencies.

Reply:
> at least 2.6.10 having the latest security fixes included

According to https://typo3.org/security/advisory/typo3-core-sa-2020-009/, that's not only fixed in 2.6.10 but also in patch releases for older minor versions of typo3fluid/fluid:

> Update to versions 2.0.8, 2.1.7, 2.2.4, 2.3.7, 2.4.4, 2.5.11 or 2.6.10 of the underlying standalone *typo3fluid/fluid* package.

Reply:
Okay fine, so let's skip TYPO3-CORE-SA-2020-009 for this PR

Reply:
you should not skip it entirely. You should create the advisory for the `typo3fluid/fluid` package instead.